examples/Lotka_Volterra_python/Lotka_Volterra_PEtab/measurement_table.tsv

```diff
@@ -0,0 +1,22 @@
+observableId	simulationConditionId	measurement	time	noiseParameters
+prey_measured	condition1	0.37035631	0	0.1
+prey_measured	condition1	0.09431187	0.1	0.1
+prey_measured	condition1	-0.2508260	0.2	0.1
+prey_measured	condition1	-0.5392827	0.3	0.1
+prey_measured	condition1	-0.7565983	0.4	0.1
+prey_measured	condition1	-0.8836758	0.5	0.1
+prey_measured	condition1	-1.0105481	0.6	0.1
+prey_measured	condition1	-1.0434417	0.7	0.1
+prey_measured	condition1	-0.9403906	0.8	0.1
+prey_measured	condition1	-1.0680282	0.9	0.1

```

Comment:
Negative values? If generated with the PEtab synthetic data methods, can be fixed by using the `fix_simulate` branch of PEtab and calling e.g. `simulator.add_noise(synthetic_data_df, zero_bounded=True)`. Or just manually :see_no_evil:

Reply:
This should be, since the observable is the `log10` of the actual value :)